projects/core/src/site-context/facade/base-site.service.ts

```diff
@@ -23,21 +23,29 @@ export class BaseSiteService implements SiteContext<string> {
    */
   getActive(): Observable<string> {
     return this.store.pipe(
-      select(SiteContextSelectors.getActiveBaseSite),
+      select(SiteContextSelectors.getActiveBaseSiteUid),
       filter((active) => Boolean(active))
     );
   }
 
   /**
    * We currently don't support switching baseSite at run time
```

Comment:
This message is not related to your PR, but looks strange here. Can we move that on top of the setActive  method instead?